examples/functions/deckPlans/DeckPlans-Example.xml

```diff
@@ -0,0 +1,1751 @@
+@@@@@@@@@    @@@@@@@@
+@ 1A              1D@
+@ 3A              3D@
+@ 4A            [4D]@
+@@@@@@@@@    @@@@@@@@
+/ Exit        Stairs@
+\ Door      to Upper@
+@@@@@@@@@    @@@@@@@@
+@ 5A 5B @ 	@ 5C 5D@
+@ 6A 6B @    @ 6C 6D@
+@ 7A 7B @    @ 7C 7D@
+@ 8A 8B @ 8E @ 8C 8D@
+@@@@@@@@@@@@@@@@@@@@@
+@@@@ lower Back @@@@@
+
+    Th upper deck has 10 rows  bench seats 2 x  2 , with a gap  for the stairhead
+    The seats on the top deck have seat sensors ( so that can show upstairs seating available)
+
+@@@ Upper Front @@@@@
+@@@@@@@@@@@@@@@@@@@@@
+@01A 01B@ 	@021 01D@
+@02A 02B@ 	@02C 02D@
+@03A 03B@ 	@03C 03D@
+@@@@@@@@@   @@@@@@@@@
+@04A 04B@     Stairs@
+@05A 05B@   to lower@
+@@@@@@@@@   @@@@@@@@@
+@06A 06B@   @06C 06D@
+@07A 07B@   @07C 07D@
+@08A 08B@   @08C 08D@
+@09A 09B@   @09C 09D@
+@10A 10B@10E@10C 10D@
+@@@@@@@@@@@@@@@@@@@@@
+@@@@Upper Back@@@@@@@
+
+     DECK PLAN (Doouble decker bus)
+         DECK (lower)           
+            OTHERSPACE (Driver)
+            PASSENGER SPACE
+              DECK ENTRANCEs (Front, Rear, Stairwell)
+                ENTRANCE EQUIPMENT
+                ENTRANCE SENSOR
+              PASSENGER SPOTs (x25)
+ 
+			  SPOT ROWs (x8)
+			  SPot COlumns (x5)
+            OTHER SPACE (Stairwell)
+        DECK (upper)                
+            PASSENGER SPACE
+               DECK ENTRANCE ( Stairwell)
+               PASSENGER SPOTs (x41)
+                 SEAT EQUIPMENT
+                 SPOT SENSOR  
+			  SPOT ROWs (x10)
+			  SPot COlumns (x5)
+
+(C) CEN 2023
+-->
+	<PublicationTimestamp>2020-12-17T09:30:47.0Z</PublicationTimestamp>
+	<ParticipantRef>SYS001</ParticipantRef>
+	<!-- ======WHAT WAS REQUESTED ========== -->
+	<PublicationRequest version="1.0">
+		<RequestTimestamp>2020-12-17T09:30:46.0Z</RequestTimestamp>
+		<ParticipantRef>SYS002</ParticipantRef>
+		<Description>Request for Erebus 1 tariff</Description>
+		<topics>
+			<NetworkFrameTopic>
+				<selectionValidityConditions>
+					<AvailabilityCondition version="any" id="r1">
+						<FromDate>2020-01-01T00:00:00Z</FromDate>
+					</AvailabilityCondition>
+				</selectionValidityConditions>
+				<NetworkFilterByValue>
+					<objectReferences>
+						<OperatorRef version="any" ref="noc:erebus"/>
+					</objectReferences>
+				</NetworkFilterByValue>
+			</NetworkFrameTopic>
+		</topics>
+	</PublicationRequest>
+	<PublicationRefreshInterval>P1M</PublicationRefreshInterval>
+	<Description>Example of DeckPlans</Description>
+	<!-- =============== RESULTS =========== -->
+	<dataObjects>
+		<CompositeFrame version="1.0" id="mb:deck_plan_example" dataSourceRef="mb:erebus" responsibilitySetRef="mb:tariffs">
+			<ValidBetween>
+				<FromDate>2023-01-01T00:00:00</FromDate>
+				<ToDate>2023-12-31T12:00:00</ToDate>
+			</ValidBetween>
+			<Name>Deck Plan    Example</Name>
+			<Description>This is an example showing how one might encode a Deck plan for a double decker bus in NeTEx. It includes seat  an, entrance and sensor equipment.</Description>
```

Comment:
Double spacing "seat  an". Might be an "and" but it should be just a comma.

Reply:
done